MCDataProducts/inc/StrawDigiMC.hh

```diff
@@ -23,18 +23,9 @@
 #include <string>
 
 // Mu2e includes
-#include <Offline/GeneralUtilities/inc/EnumToStringSparse.hh>
+#include <Offline/MCDataProducts/inc/DigiProvenance.hh>
```

Comment:
I recognize that you are following the mode of existing code here but I would like you to make one more change.

My understanding of the C++ standard is that we should use the angle bracket syntax for system-defined and compiler-defined headers and that we should use the quote syntax for headers to be resolved from the -I command line options.   There is a difference.  While that difference is not likely to matter for our code for our compilers I would like to do it right.

There are several other files with this same issue.

Reply:
The model I've always used is "quotes for in-directory includes, and brackets for search-path includes." The standard is 200 bucks but I interpret ChatGPT as being on your side, so I've updated all of the files touched by this PR to use the quotes.

Reply:
Thanks for making the change.  While the standard costs real money, the drafts don't !  This is a recent draft of the upcoming 2023 standard.  https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2023/n4950.pdf  .  In section 15.3 p 466, 
 
> An implementation can provide a mechanism for making arbitrary source files available to the < > search.
>   However, using the < > form for headers provided with the implementation and the " " form for sources outside the  control of the implementation achieves wider portability. For instance:
>   #include <stdio.h>
>   #include <unistd.h>
>   #include "usefullib.h"
>   #include "myprog.h"

( A former member of the international standards committee has an office 2 doors down from me.  ).